services/server/test/helpers/ServerFixture.ts

```diff
@@ -82,8 +136,8 @@ export class ServerFixture {
 
     beforeEach(async () => {
       rimraf.sync(this.server.repository);
-      await resetDatabase(this.storageService);
-      console.log("Resetting the StorageService");
+      await resetDatabase(this.sourcifyDatabase);
+      console.log("Resetting SourcifyDatabase");
     });
```

Comment:
Why just resetting the database? Aren't there tests that need this for FS based storage?